utils/common/src/all.records.ts

```diff
@@ -186,20 +186,22 @@ export async function updateAllRecords(
 export async function createAllRecords(
   sheetId: string,
   records: Flatfile.RecordData[],
-  tick?: TickHelper
+  tick?: TickHelper,
+  customRecordsPageSize: number = DEFAULT_PAGE_SIZE
```

Comment:
Just to make is easier to add more options later:
```suggestion
  options: {pageSize: number = DEFAULT_PAGE_SIZE}
```